src/LogViewer/chunk-worker.js

```diff
@@ -10,6 +10,22 @@ const LINE_CHUNK = 1000;
 const DECODER = new TextDecoder('utf-8');
 // Setting a hard limit on lines since browser have trouble with heights starting at around 16.7 million pixels and up
 const CHUNK_LIMIT = 883000 / LINE_CHUNK;
+// HTML Escape helper utility
+const util = (() => {
```

Comment:
I think this makes the escaping a little more abstract than necessary. I think we can change it like so to simplify:

```js
const ESCAPE_REGEX = /[&<>'"]/g;
const ESCAPE_FUNCTION = entity => ESCAPE_ENTITY[entity];
const ESCAPE_ENTITY = {
  '&': '&amp;',
  '<': '&lt;',
  '>': '&gt;',
  "'": '&#39;',
  '"': '&quot;',
  '`': '&#x60;'
};
```